sys/shell/commands/sc_openwsn.c

```diff
@@ -0,0 +1,569 @@
+{
+    (void)arg;
+
+    for (int i = 0; i < MAXNUMNEIGHBORS; i++) {
+        netif_addr_to_str(neighbors_vars.neighbors[i].addr_64b.addr_64b, 8,
+                          _addr_str);
+        if (memcmp(_addr_str, "00:00:00:00:00:00:00:00", 8) != 0) {
+            printf("%02i. %s\n", i, _addr_str);
+        }
+    }
+    return 0;
+}
+
+static void _print_cell_usage(void)
+{
+    puts("Usage:");
+    puts("\tcell list: show all active cell");
+    puts(
+        "\tcell add <slot_offset> <channel_offset> <adv|tx|rx> [<address>]: add cell directly to schedule");
+    puts(
+        "\tcell rmv <slot_offset> <channel_offset> <adv|tx|rx> [<address>]: remove cell directly in schedule ");
+}
+
+static int _cell_list_cmd(char *arg)
+{
+    (void)arg;
+
+    for (int i = 0; i < MAXACTIVESLOTS; i++) {
+        switch (schedule_vars.scheduleBuf[i].type) {
+            case CELLTYPE_TX:
+                printf("neigh: %s, slot: %03i, chan: %02i, type: TX\n", \
+                       netif_addr_to_str(
+                           schedule_vars.scheduleBuf[i].neighbor.addr_64b,
+                           8, _addr_str),
+                       schedule_vars.scheduleBuf[i].slotOffset, \
+                       schedule_vars.scheduleBuf[i].channelOffset);
+                break;
+            case CELLTYPE_RX:
+                printf("slot: %03i, chan: %02i, type: RX\n", \
+                       schedule_vars.scheduleBuf[i].slotOffset, \
+                       schedule_vars.scheduleBuf[i].channelOffset);
+                break;
+            case CELLTYPE_TXRX:
+                printf("neigh: %s, slot: %03i, chan: %02i, type: RXTX\n", \
+                       netif_addr_to_str(
+                           schedule_vars.scheduleBuf[i].neighbor.addr_64b,
+                           8, _addr_str),
+                       schedule_vars.scheduleBuf[i].slotOffset, \
+                       schedule_vars.scheduleBuf[i].channelOffset);
+                break;
+            default:
+                break;
+        }
+    }
+
+    return 0;
+}
+
+static int _cell_manage_cmd(int argc, char **argv)
+{
+    open_addr_t addr;
+
+    memset(&addr, 0, sizeof(addr));
+    cellType_t type;
+    int res;
+    bool add;
+
+    if (argc < 5) {
+        _print_cell_usage();
+        return -1;
+    }
+
+    if (argc == 6) {
+        addr.type = ADDR_64B;
+        size_t len = netif_addr_from_str(argv[5], addr.addr_64b);
+        if (len == 0) {
+            puts("Error: invalid address");
+            return -1;
+        }
+    }
+    else {
+        addr.type = ADDR_ANYCAST;
+    }
+
+    add = strcmp(argv[1], "add") == 0 ? true : false;
+
+    if (strcmp(argv[4], "adv") == 0) {
+        type = CELLTYPE_TXRX;
+    }
+    else if (strcmp(argv[4], "tx") == 0) {
+        type = CELLTYPE_TX;
+    }
+    else if (strcmp(argv[4], "rx") == 0) {
+        type = CELLTYPE_RX;
+    }
+    else {
+        _print_cell_usage();
+        return -1;
+    }
+
+    if (add) {
```

Comment:
```suggestion
    if (strcmp(argv[1], "add") == 0) {
```
And drop the `add` boolean.